src/Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs

```diff
@@ -81,12 +81,18 @@ public static class LoggingConfiguration
 
         public static void RegisterFeatureNamespace<T>(string key)
         {
-            KeyCategories[key] = typeof(T).Namespace + ".*";
+            lock (KeyCategories)
```

Comment:
Why not creating a lockobject like we do in other places in the codebase?

Reply:
The `KeyCategories` class is private.

Reply:
Ok, fair point.